packages/block-editor/src/components/grid-visualizer/grid-item-resizer.js

```diff
@@ -28,15 +104,41 @@ export function GridItemResizer( { clientId, onChange } ) {
 					height: '100%',
 				} }
 				enable={ {
-					bottom: true,
+					bottom: enableBottom,
 					bottomLeft: false,
 					bottomRight: false,
-					left: false,
-					right: true,
-					top: false,
+					left: enableLeft,
+					right: enableRight,
+					top: enableTop,
 					topLeft: false,
 					topRight: false,
 				} }
+				bounds={ boundingElement }
+				boundsByDirection
+				onResizeStart={ ( event, direction ) => {
+					/*
+					 * The container justification and alignment need to be set
+					 * according to the direction the resizer is being dragged in,
+					 * so that it resizes in the right direction.
+					 */
+					setResizeDirection( direction );
+
+					/*
+					 * The mouseup event on the resize handle doesn't trigger if the mouse
+					 * isn't directly above the handle, so we try to detect if it happens
+					 * outside the grid and dispatch a mouseup event on the handle.
+					 */
```

Comment:
Should this fix live in `ResizableBox` so that future users of the component don't run into the same issue?

Reply:
Hmm good question. This issue only surfaces when we set `bounds` on the box, so it's not really useful for all the instances. And this fix depends on knowing about a parent element, which `ResizableBox` doesn't. It doesn't even implement its own `onResizeStart`, just passes it directly to `Resizable`. So I'm not sure it's worth adding complexity to the component for what so far is a pretty niche use case. Maybe worth doing it if similar instances pop up in other places?

Reply:
The `mouseup` handler could be added to `ownerWindow` or `ownerDocument` to avoid it having to know about a parent element.

But yeah, not sure if it's needed in `ResizableBox`, up to you.

Reply:
Hmmm not sure that'll work if we have the iframe in the way. The parent element I added the event listener to is in the actual editor canvas, but the component isn't.